### PR TITLE
Vignesh/mistral tt implementation set 2

### PR DESCRIPTION
**Pull request by vigneshkeerthivasanx**
This branch contains 

- fix for low PCC due to feed-forward submodule.
- Update rotary embedding to use complex_multiplication in TT ops

### CODE REVIEW

**Review by tt-aho — CHANGES_REQUESTED** [review]
Can you fix your git history/commits for this PR? It is showing commits from main as part of your changes which makes it hard to review.

Am also blocking for now as there are still ongoing complex tensor requests/discussions in #3412 to finish discussing/resolving

**Review by tt-aho — COMMENTED** [review]
lgtm overall, will approve after complex tensor changes are finalized

**Review by farbabi — APPROVED** [review]

**Review by tt-aho — APPROVED** [review]
Approving now since #3412 comments have been mostly addressed so lgtm once integrated with new changes

**Pull request merged**
